docs/tutorials/extending_admin_ui/1_creating_a_dashboard_tab.md

```diff
@@ -1,7 +1,8 @@
 # Step 1 - Creating a Dashboard tab
 
 The Dashboard is the front page that you visit after logging in to the Back Office.
-By default it contains two blocks: "My content" and "Common content", which list Content items and Media.
+You can also get to it from any other page by selecting eZ logo in the top left corner.
```

Comment:
```suggestion
You can also get to it from any other page by selecting the site in the top left corner.
```
Safer option, in case a customer has customized the BO to use their own logo